openvdb/doc/changes.txt

```diff
@@ -17,6 +17,8 @@ Improvements:
   @vdblink::tree::LeafNode::modifyValueAndActiveState()
   LeafNode::modifyValueAndActiveState@endlink now modify voxel values
   in place for improved performance.
+- Add CMake support for linking against Jemalloc and TBB malloc and enable
+  Jemalloc by default.
```

Comment:
Should probably add windows/mac notes here

Reply:
Done.